### PR TITLE
search bar with search filter

### PR DESCRIPTION
**Pull request by CathyF9600**

### CODE REVIEW

**Comment by RobertRen1122** [comment]
We removed the search button and added a filter. (with 5 links to different sites included)

File: frontend/package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "private": true,
   "homepage": "/",
   "dependencies": {
+    "@material-ui/core": "^4.12.1",
```
Comment (Louis-He) [inline frontend/package.json]: unnecessary dependency

File: frontend/package.json [hunk 2]
```diff
@@ -39,5 +40,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "@material-ui/icons": "^4.11.2"
```
Comment (Louis-He) [inline frontend/package.json]: Please use the bootstrap icons instead. No mixed use of bootstrap and material

**Review by Louis-He — CHANGES_REQUESTED** [review]

**Comment by RobertRen1122** [comment]
changes reviewed

**Comment by RobertRen1122** [comment]
Trying carousel, closing this branch

**Pull request closed without merging**